SearchResults.js

```diff
@@ -102,42 +105,75 @@ function SearchResults() {
   self.loading = ko.pureComputed(function () { self.activeRequests() == 0; });
   self.uninitialised = ko.pureComputed(function () { return self.pullRequests().length == 0; });
 
+
+  function loadAllPages(query, onComplete) {
+    var totalCount;
+    var pullRequests = [];
+    // Function to load a page of results
+    function getPage(uri) {
+      self.activeRequests(self.activeRequests() + 1);
+      $.getJSON(uri, function (data, textStatus, jqXHR) {
+        self.errorMessage(null);
+        totalCount = data.total_count;
+        // Add the pull requests to the existing cache
+        pullRequests = pullRequests.concat(data.items);
+        // Get the link to the next page of results
+        var nextLinkSuffix = "; rel=\"next\"";
+        var linksHeader = jqXHR.getResponseHeader("Link");
+        if (linksHeader == null)
+          linksHeader = "";
+        var nextLinks = linksHeader.split(",").filter(function (link) { return link.endsWith(nextLinkSuffix); });
+        if (nextLinks.length > 0) {
+          var nextLink = nextLinks[0];
+          nextLink = nextLink.substring(1, nextLink.length - nextLinkSuffix.length - 1);
+          getPage(nextLink);
+        } else
```

Comment:
You've got quite a lot of code to parse the link header. There is a utility on npm to parse-link-header, but to use it we'd need to refactor the whole project to use webpack or something.